care/facility/models/mixins/permissions/facility.py

```diff
@@ -57,9 +57,9 @@ def has_object_write_permission(self, request):
             or request.user.user_type == User.TYPE_VALUE_MAP["StaffReadOnly"]
         ):
             return False
-        if request.user.user_type < User.TYPE_VALUE_MAP["Staff"]:  # todo Temporary
-            return False
-        return self.has_object_read_permission(request)
+        if request.user.user_type >= User.TYPE_VALUE_MAP["DistrictAdmin"]:
+            return self.has_object_read_permission(request)
```

Comment:
staff also needs to be checked with the `self.has_object_read_permission(request)` since staff can also have access to a facility

Reply:
Staff doesn't have write permission, right? According to the issue description, all users below the DistrictAdmin level can only edit their home facility (including the staff).  So I am checking the admin's access inside the if condition. Rest are being checked for their home facility.

Reply:
you can keep the existing permissions, staff can edit facilities that they have access to

Reply:
Updated it. Staff now have write permission to all their linked facilities. Also wanted to confirm if similar roles like the WardAdmin or LocalBodyAdmin require these additional write permissions.

CC: @gigincg